### DISCUSSION TITLE
GY-302 BH1750 Helligkeitssensor an ESP8266-01S mit Tasmota betreiben

### DISCUSSION THREAD

**Discussion by ich77** [arendst/Tasmota#21823] [Support request] [answered]
Hallo zusammen,
ich versuche den oben genannten Helligkeitssensor über i2c an einem ESP8266 zu betreiben. Leider komme ich damit nicht weiter, ich bekomme es nicht zum laufen. Ich sehe das Gerät über i2cscan auf der Adresse 0x23 kann aber den Treiber mit I2cDriver11 nicht ändern, habe immer nur I2cDriver7
Hat jemand eine Idee, wie ich weiterkomme?
Danke

**Comment by sfromis** [comment, marked as answer]
You need a [build](https://tasmota.github.io/docs/BUILDS/) including support for the sensor, the I2CDriver command **only** lists the ones which are included. Upgrading to `tasmota-sensors.bin.gz` should provide this.

**Comment by HofeBY** [comment]
Versuche mal
I2CDriver7 0
gefolgt von
I2CDriver
Ergebnis hier posten.

**Reply by ich77** [reply]
Vielen Dank, hier das Ergebnis:
```

14:27:50.599 CMD: I2CDriver
14:27:50.604 MQT: stat/tasmota_B0F9D3/RESULT = {"I2CDriver":"!7"}
14:27:52.305 QPC: Reset
14:27:55.253 MQT: tele/tasmota_B0F9D3/STATE = {"Time":"2024-07-25T14:27:55","Uptime":"0T00:00:11","UptimeSec":11,"Heap":26,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":1,"Wifi":{"AP":1,"SSId":"ACCESS","BSSId":"0C:72:74:3A:ED:AA","Channel":6,"Mode":"11n","RSSI":48,"Signal":-76,"LinkCount":1,"Downtime":"0T00:00:05"}}
```

**Reply by sfromis** [reply]
You need to follow my suggestion of using a binary including support for the sensor.

**Reply by ich77** [reply]
@sfromis Thank you very much! That worked! Now the sensor shows the illuminance!

**Reply by HofeBY** [reply]
Das hatte ich befürchtet, das der I2CDriver11 garnicht enthalten ist.

Hier ein Link zu selber compilieren eigener Tasmota version.
Ich ändere die user_config_override.h so, das auch der Treiber mit einkompiliert wird.
https://github.com/HofeBY/Tasmota-Firmware
Halte Dich an die Liesmich-zuerst.txt
Die vorhandenen *.bin Files enthalten den Treiber noch nicht !

Oberfränkische Grüße
HofeBY

**Comment by borsti67** [comment]
Hallo @ich77,
liefert der Sensor bei Dir mit dieser Firmware brauchbare Werte?
Ich habe jetzt 2 ausprobiert (an WEMOS D1 Mini), beide liefern Werte, die um den Faktor 10 oder eher 100 zu niedrig sind - bei Tageslicht so um die 80, bei Dämmerung 2-7...
